joss-paper/Makefile

```diff
@@ -0,0 +1,58 @@
+# =============================================================================
+# @file    Makefile
+# @brief   Makefile for generating previews of the paper
+# @author  Michael Hucka <mhucka@caltech.edu>
+# @license Please see the file named LICENSE in the project directory
+# @website https://github.com/casics/dassie
+# =============================================================================
+
+# Change the following values to match your configuration.
+# .............................................................................
+
+input	:= paper
+
+vol     := 3
+issue   := 28
+gh_issue  := 850
+year    := 2018
+submitted := 23 July $(year)
```

Comment:
Should we change this? Or it doesn't matter?

Reply:
I'll change this when it's closer to submission, so I actually know the date :-) Note: the Makefile is not actually part of the submission, it's just a way for us to make a PDF from the markdown (which happens automatically in the JOSS review) to eventually submit to arXiv.